node/src/Transport.ts

```diff
@@ -1,4 +1,5 @@
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID } from 'node:crypto';
+
```

Comment:
No reason for this blank line.

Reply:
I use to separate import types, it's easier to read. I can remove it too if you want.

Reply:
No file in mediasoup TypeScript code separates system and local imports with a blank line. Please, let's always honor the project style rather than trying to introduce our preferred one.